src/app/views/query-response/response/ResponseMessages.tsx

```diff
@@ -119,7 +119,7 @@ export const ResponseMessages = () => {
   ) {
     messageBars.push(
       <div key={'displayMessage'}>
-        <MessageBar intent='warning'>
+        <MessageBar intent='info'>
```

Comment:
Let's leave this one as a warning as it was previously 
![image](https://github.com/user-attachments/assets/7c563976-3eab-4fe8-84a0-5a527f7b352d)